index.js

```diff
@@ -82,6 +86,15 @@ class ServerlessCustomDomain {
     this.targetHostedZoneName = this.givenDomainName.substring(this.givenDomainName.indexOf('.') + 1);
   }
 
+  setEndpointType(endpointType = 'EDGE') {
+    if (!SUPPORTED_ENDPOINT_TYPES.includes(endpointType)) throw new Error(`${endpointType} is not supported endpointType, use EDGE or REGIONAL.`);
```

Comment:
Do the endpoint types need to be capitalized? Serverless seems to standardize on lowercase wherever possible